packages/style-utilities/src/styles/CommonStyles.ts

```diff
@@ -23,6 +25,16 @@ export function getScreenSelector(min: number, max: number): string {
   return `@media only screen and (min-width: ${min}px) and (max-width: ${max}px)`;
 }
 
+/**
+ * The style which turns off high contrast adjustment in browsers.
+ */
+export function getHighContrastAdjustSelector(): IRawStyle {
```

Comment:
It would be more accurate to call this `getHighContrastAdjustStyle`.

Reply:
Or `getNoHighContrastAdjustStyle`?

Reply:
I think it should be `getHighContrastNoAdjustStyle` (with "No" before adjust)

Reply:
Thank you both, naming is always hard so suggestions are helpful! Will update